models/yang/sonic/sonic-config-mgmt.yang

```diff
@@ -0,0 +1,39 @@
+module sonic-config-mgmt {
+  namespace "http://github.com/Azure/sonic-config-mgmt";
+  prefix scm;
+
+  organization
+    "BRCM";
+  contact
+    "BRCM";
+  description
+    "SONIC ERROR";
+
+  revision 2019-09-15 {
+    description
+      "Initial revision.";
+  }
+
+
+  rpc copy {
+      input {
+          leaf source {
```

Comment:
Please do add a description to source and destination attributes.

Reply:
Done.